### ISSUE TITLE
Email notifications not compatible with special characters?

### ISSUE THREAD

**Issue by alvarogois** [jrtashjian/omniform#41]
I'm getting the email notifications with unicode instead of special characters for the Portuguese language, like õ, ç, ã, etc. Is this expected or there's something I need to do to fix it?

Thanks.

**Comment by jrtashjian** [comment]
Thanks for reporting this! I’m taking a look and will follow up soon.

**Comment by alvarogois** [comment]
Anything I can do to test for possible issues?

Thanks.

**Comment by jrtashjian** [comment]
I apologize for the time it's taken me to actually look at this issue.

I was able to reproduce the problem in [v1.2.1](https://github.com/jrtashjian/omniform/releases/tag/v1.2.1), but the latest code on the `master` branch has fixed this issue. I'm currently preparing the latest code for release before the end of the year.

**Comment by alvarogois** [comment]
Thanks. That would be great.

**Comment by jrtashjian** [comment]
So it looks like I only partly solved the issue. I ran across the problem when viewing the responses in the admin, not just from the email notification. I believe I've resolved it now.

![image](https://github.com/user-attachments/assets/3ecca25d-5e95-482e-9e10-909103733ece)

**Comment by alvarogois** [comment]
Yes, I see the issue also when viewing the responses in the admin. Should I test the updated code? Thanks.

**Comment by jrtashjian** [comment]
I’ve just released version 1.3.1 of the plugin, which should resolve the issue with special characters.

**Comment by jrtashjian** [comment]
Marking as resolved: https://wordpress.org/support/topic/problem-with-special-characters-12/#post-18263992
